docs/src/cargo-toml-configuration.md

```diff
@@ -9,6 +9,17 @@ the `--dev`, `--profiling`, and `--release` flags passed to `wasm-pack build`.
 The available configuration options and their default values are shown below:
 
 ```toml
+[package.metadata.wasm-pack.profile.dev]
+# Should `wasm-opt` be used to further optimize the wasm binary generated after
+# the Rust compiler has finished? Using `wasm-opt` can often further decrease
+# binary size or do clever tricks that haven't made their way into LLVM yet.
+#
+# Configuration can be set to `false` if you want to disable it (as is the
+# default for the dev profile), or it can be an array of strings which are
+# explicit arguments to pass to `wasm-opt`. For example `['-Os']` would optimize
+# for size while `['-O4']` would execute very expensive optimizations passes
```

Comment:
This is missing "size" and "speed" meta options [that we planned in the original design](https://github.com/rustwasm/wasm-pack/issues/159#issuecomment-454888890):

> ```toml
> # This is the dev profile, but could also be the profiling or release profiles.
> [package.metadata.wasm-pack.profile.dev]
> # The `wasm-opt` key may be absent, in which case we choose a default
> #
> # or we can explicitly configure that we *don't* want to run it
> wasm-opt = false
> # or use our default alias to optimize for size
> wasm-opt = "size"
> # or use our default alias to optimize for speed
> wasm-opt = "speed"
> # or give your own custom set of CLI flags
> wasm-opt = ["--dce", "--duplicate-function-elimination", "--instrument-memory"]
> ```

Is the intention to add support for these in follow up PRs?

Reply:
True! I forgot to mention that here, but I actually first implemented it as a string being a space-separated list of arguments (like `wasm-opt = "-Os -some-other-pass"`), and then I read the original issue and liked your idea better.

I figured though it may be best to do that as a separate PR as it should be pretty easy to add on top of this, but rather we'd just want to get the bare bones in here